lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -98,7 +100,7 @@ protected function _getPortableTableIndexesList($tableIndexes, $tableName = null
             $keyName = strtolower($tableIndex['name']);
             $buffer  = [];
 
-            if (strtolower($tableIndex['is_primary']) === 'p') {
+            if ($tableIndex['is_primary'] === 'P' || $tableIndex['is_primary'] === 'p') {
```

Comment:
Just `'P'` will be enough. On [wnameless/oracle-xe-11g](https://hub.docker.com/r/wnameless/oracle-xe-11g), 
```sql
SELECT DISTINCT constraint_type FROM user_constraints

-- +---------------+
-- |CONSTRAINT_TYPE|
-- +---------------+
-- |U              |
-- |R              |
-- |P              |
-- |C              |
-- |O              |
-- +---------------+
```